docs/GraFx-Studio/concepts/conversion/index.md

```diff
@@ -0,0 +1,37 @@
+# Conversion of Desktop files
+
+## Intro
+
+Imagine the scenario where you have a design created in a Desktop application. This file should be the basis for your Smart Template.
+
+AN easy design could be recreated in GraFx Studio, for a more complex file it would be handy to have a conversion from your existing file to GraFx Studio.
+
+That is wat a conversion does.
+
+## Intermediate File Format
+
+To accomodate several application, we designed the process to enable Desktop applications to export to an Intermediate File Format.
```

Comment:
applications

Reply:
a bit weird sentence maybe (we designed the process to...): We developed plugins for several desktop applications to easily export your design to an Intermediate File Format.